lib/github_client.rb

```diff
@@ -0,0 +1,24 @@
+require 'net/http'
+require 'json'
+
+class GitHubClient
```

Comment:
Why not use the `octokit` gem provided by GitHub? 🤔  Not worth it for so little API surface we need?

Reply:
Yep, that's exactly it (as per 1e17a70's description).

If we'll need to do more advanced requests or processing with GitHub in the future, `octokit` would be my choice.